website/user-documentation/using-gqcp.md

```diff
@@ -0,0 +1,76 @@
+
+# Using GQCP(y)
+
+After you have installed GQCP, you can either interface directly with GQCP [through C++](#using-gqcp-through-cpp) or use its [Python bindings](#using-gqcp-through-python-bindings-gqcpy).
+
+## Using GQCP through CPP
+
+### Hello GQCP: source file
+
+Given the following XYZ file
+
+```bash
+3
+
+O          0.00000       -0.07579        0.00000
+H          0.86681        0.60144        0.00000
+H         -0.86681        0.60144        0.00000
+```
+
+a prototypical `Hello GQCP` driver is given by
```

Comment:
```suggestion
a prototypical `Hello GQCP` driver `driver.cpp` is given by
```

Reply:
I've added the explicit name for the driver file, otherwise first-time users might be confused.